.changeset/modern-worlds-fry.md

```diff
@@ -16,7 +16,7 @@
 "@fluidframework/test-utils": major
 ---
 
-provideEntryPoint is required
+container-runtime: provideEntryPoint is required
```

Comment:
I'd say `test-utils` here - yes it's about container-runtime functionality, but this changeset is about constructors on test util classes (I believe)

Reply:
```suggestion
test-utils: provideEntryPoint is required
```